src/components/AssetMatching/components/SectionHeader/SectionHeader.tsx

```diff
@@ -14,21 +14,33 @@ interface Props {
   score?: string | number;
   matchingActions?: MatchingAction[];
   styleType: formFieldStyles;
+  actionButtonFill?: boolean;
+  customCheckboxContent?: JSX.Element;
```

Comment:
as this is JSX i would expect capital case

Reply:
Should we use capital case in variables that hold jsx element? 
I thought only functional components should start with capital letter.

for example: 
```
we use button like this:
<Button iconLeft={...} />

iconLeft holds a jsx element.
```

Reply:
sorry my fault you are right only for components